x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_connect/engine_api_integration.tsx

```diff
@@ -13,31 +13,45 @@ import { compressToEncodedURIComponent } from 'lz-string';
 import {
   EuiCodeBlock,
   EuiFlexGroup,
+  EuiFlexItem,
   EuiLink,
   EuiSpacer,
   EuiTab,
   EuiTabs,
   EuiText,
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+
 import { FormattedMessage } from '@kbn/i18n-react';
 
 import { useCloudDetails } from '../../../../shared/cloud_details/cloud_details';
+import { docLinks } from '../../../../shared/doc_links';
 import { KibanaLogic } from '../../../../shared/kibana';
 import { EngineViewLogic } from '../engine_view_logic';
 
 import { EngineApiLogic } from './engine_api_logic';
 
 import { elasticsearchUrl } from './search_application_api';
 
-const SearchUISnippet = (esUrl: string, engineName: string, apiKey: string) => `
-import EnginesAPIConnector from "@elastic/search-ui-engines-connector";
+const clientSnippet = (esUrl: string, searchApplicationName: string, apiKey: string) => `
+import Client from '@elastic/search-application-client'
+// or through CDN
+// const Client = window['SearchApplicationClient']
+
+const request = Client(
+  ${searchApplicationName},
+  ${esUrl},
+  ${apiKey},
```

Comment:
suggestion: apiKey shows up as null here when there isn't an apiKey. have it fallback like the other examples.

Also double check generating an apiKey, it is replaced :)